web/src/components/onboarding/IndustryDropdown.test.tsx

```diff
@@ -10,4 +11,40 @@ describe("Industry Dropdown", () => {
 
     expect(within(items[0]).getByTestId("generic")).toBeInTheDocument();
   });
+
+  it("displays the generic industry as the single option when there is no industry match", async () => {
+    render(<IndustryDropdown />);
+    const searchTerm = `some-industry-${randomInt()}`;
+
+    fireEvent.click(screen.getByLabelText("Industry"));
+    fireEvent.change(screen.getByLabelText("Industry"), {
+      target: { value: searchTerm },
+    });
+
+    fireEvent.click(screen.getByLabelText("Industry"));
+    expect(screen.getByTestId("generic")).toBeInTheDocument();
+    expect(screen.getAllByText(searchTerm).length).toEqual(1);
+    expect(screen.queryByTestId("certified-public-accountant")).not.toBeInTheDocument();
+  });
+
+  it("updates the value of the industry dropdown input when the no-matching industries button is clicked", () => {
+    render(<IndustryDropdown />);
+
+    const inputElement = screen.getByLabelText("Industry");
+    fireEvent.click(inputElement);
+    fireEvent.change(inputElement, { target: { value: "ac5" } });
+    fireEvent.click(screen.getByTestId("generic"));
+
+    expect((screen.getByLabelText("Industry") as HTMLInputElement).value).toBe("All Other Businesses");
+  });
+
+  it("displays the user input and a message that the input falls under the industries below", () => {
```

Comment:
two small notes here about naming tests:
- You say "displays the user input" but I don't see anything in the test that appears to be asserting about the user input.
- You describe the message it displays as "a message that the input falls under the industries below". This is fine for now, because it is what that string says. But that is configurable content, which means it could change at any time. 

We want to write robust tests that will accurately describe behavior so that future developers can read the tests to learn about the intended behavior of a component. So good naming is important. 

Because of that, I think we should remove clauses from the test name that are not actually being tested.

Similarly, we should avoid referencing the specific value of the message (like how it says "falls under industries below") because that's liable to change. In fact, for this case it's very likely to change almost immediately because I know Jess & Ce wanted to make it more generic, because sometimes the options are displayed above, not below. Choosing a good, descriptive name for what _we_ as developers call this message, and then being consistent about it, is important to keeping our code cohesive and understandable. 

Right now we have three disparate ways of referring to this message:
- In config, it is `mappedIndustryCopy`
- In the data-testid, it is `styled-search-term`
- In the test name, it is "a message that the input falls under the industries below"

Try to think of a descriptive, concise name that communicates what this message represents. And then we should use that same name to refer to it everywhere in the code.